clang/include/clang/CodeGen/CodeGenABITypes.h

```diff
@@ -104,6 +104,12 @@ llvm::Type *convertTypeForMemory(CodeGenModule &CGM, QualType T);
 unsigned getLLVMFieldNumber(CodeGenModule &CGM,
                             const RecordDecl *RD, const FieldDecl *FD);
 
+/// Return a signed constant pointer.
+llvm::Constant *getConstantSignedPointer(CodeGenModule &CGM,
+                                         llvm::Constant *pointer,
```

Comment:
Here and in many other places - use `PascalCase` for variables (https://llvm.org/docs/CodingStandards.html#name-types-functions-variables-and-enumerators-properly). The surrounding code uses the convention, so it looks like there is no reason not to follow it.

I'll leave a bunch of similar comments, but please do not limit your fixes of variable names to my comments since I might have missed some.